docs/content/en/docs/install/install.md

```diff
@@ -102,30 +125,47 @@ modify some values, and use the modified file to install Keptn:
 
 1. Edit your local copy to modify some values
 
-1. Install Keptn by adding the following string to your `helm upgrade` command line:
+1. To apply the modified configuration.
+   install Keptn by adding the following string
+   to your `helm upgrade` command line:
 
    ```shell
    --values=values.yaml
    ```
 
-You can also use the `--set` flag
-to specify a value change for the `helm upgrade --install` command.
-Configuration options are specified using the format:
+   For example, if you create a `my.values.yaml`
+   and modified some configuration values,
+   the command sequence to apply your configuration is:
 
-```shell
---set key1=value1 \
---set key2=value2 ...
-```
+   ```shell
+   helm repo add keptn https://charts.lifecycle.keptn.sh
+   helm repo update
+   helm upgrade --install keptn keptn/keptn \
+    --values my.values.yaml \
+    -n keptn-lifecycle-toolkit-system --create-namespace --wait
+   ```
 
-The [helm-chart](https://github.com/keptn/lifecycle-toolkit-charts/blob/main/charts/keptn/README.md) page
-contains the full list of available values.
+   You can also use the `--set` flag
+   to specify a value change for the `helm upgrade --install` command.
+   Configuration options are specified using the format:
+
+   ```shell
+   --set key1=value1 \
+   --set key2=value2 ...
+   ```
 
 ## Control what components are installed
 
-Keptn consists of multiple components, each of which enables a specific use-case.
-Each of the components is packaged into a subchart of Keptn which can be configured individually.
+Keptn consists of multiple components,
+each of which enables a specific use-case.
+Each of the components is packaged into a subchart of Keptn
+which has values that can be configured for that component
+in the umbrella `values.yaml` file.
```

Comment:
This is technically correct but I think it will puzzle readers. They just learned how to configure Keptn in the previous section and now we are telling them again how to do it, but we're introducing a new concept "the umbrella value.yaml".  I would not add these details here.

Reply:
Okay, need some rewriting.  What I am trying to explain is that the values for each component are in that component's values.yaml file and described in that component's README file but, if I want to modify the value of any of those parameters, I need to copy the appropriate parmeters into the umbrella values.yaml file.  I thought I had called it Keptn umbrella in the table but it isn't there now.  I think the implementation is confusing unless one is super-seeped in Kubernetes/Helm internals -- usually I have a config file and I just go into that file and modify the value of a parameter.  So how do we succinctly explain that for the values associated with sub-charts?

Maybe we need to spell out that they need need to copy the parameter lines they want to modify from the subchart values.yaml file into the umbrella values.yaml file and reset them there?

Reply:
See if this works.  I realized that we had a structural problem with the location of the table so moved that into its own section before the sections about modifying Helm values, then gave more explicit instructions about how to modify the configuration of the components.  See if this works.